cli/types/index.d.ts

```diff
@@ -1069,6 +1069,18 @@ declare namespace Cypress {
      */
     its<K extends keyof Subject>(propertyName: K, options?: Loggable): Chainable<Subject[K]>
 
+    /**

```

Comment:
I don't *think* this comment needs to be duplicated, see `invoke` above.

Reply:
confirmed 👍